js/compiler/compiler.go

```diff
@@ -218,15 +217,7 @@ func (b *babel) Transform(logger logrus.FieldLogger, src, filename string) (stri
 	if err = b.vm.ExportTo(vO.Get("code"), &code); err != nil {
 		return code, nil, err
 	}
-	var rawMap map[string]interface{}
-	if err = b.vm.ExportTo(vO.Get("map"), &rawMap); err != nil {
-		return code, nil, err
-	}
-	var srcMap SourceMap
-	if err = mapstructure.Decode(rawMap, &srcMap); err != nil {
-		return code, &srcMap, err
-	}
-	return code, &srcMap, err
+	return code, nil, err
```

Comment:
I don't think you now return anything other than `nil` as the second result here and in `Compiler.Transform()` :confused:

Reply:
yeah, but I didn't want to remove it everywhere just to re-add it in #2082

Reply:
hmm fair enough, but at least put in a comment with a `TODO` and a link to the other PR before this line, now it looks very weird